python-sdk/src/astro/files/locations/base.py

```diff
@@ -7,9 +7,19 @@
 from urllib.parse import urlparse
 
 import smart_open
+from airflow.hooks.base import BaseHook
 
 from astro.constants import FileLocation
 
+CUSTOM_CONN_TYPE_TO_FILE_SCHEME = {
+    "bigquery": {"gs"},
+    "google_cloud_platform": {"gs", "gdrive", "https"},
```

Comment:
@sunank200 I think we don't support `https` paths as GCS path, right?

Reply:
So, this is the scenario with `gdrive`, it uses `https` paths but uses `google_cloud_platform` as connection_type

Reply:
changed the logic here: https://github.com/astronomer/astro-sdk/pull/1478/commits/181366c944d65117e7ab0d8b06b72ffc9c23a270